src/_tutorials/server/cmdline.md

```diff
@@ -465,45 +404,45 @@ You can continue to write to the file until done,
 at which time, you must close the file.
 The `close()` method is asynchronous and returns a Future.
 
-{% prettify dart %}
-IOSink quotes = new File('quotes.txt').openWrite(mode: FileMode.append);
+<?code-excerpt "misc/lib/tutorial/cmdline.dart (write quotes)"?>
+```dart
+final quotes = File('quotes.txt').openWrite(mode: FileMode.append);
 
-quotes.write('A woman is like a tea bag; ');
-quotes.write('you never know how strong it is until it\'s in hot water.');
-quotes.writeln(' -Eleanor Roosevelt');
+quotes.write("Don't cry because it's over, ");
+quotes.writeln("smile because it happened. -Dr. Seuss");
 await quotes.close();
-print('Done!');
-{% endprettify %}
+```
 
 ## Getting environment information
 
-Use the
-<a href="{{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}}/dart-io/Platform-class.html" target="_blank">Platform</a>
-class
+{% assign PlatformAPI = ioAPI | append: '/Platform' -%}
+
+Use the [Platform]({{PlatformAPI}}-class.html){:target="_blank"} class
 to get information about the machine and OS that the program is running on.
 Note: Use the Platform class from the dart:io library,
 not from the dart:html library.
 
-`Platform.environment` provides a copy of the environment
+[Platform.environment]({{PlatformAPI}}/environment.html){:target="_blank"}
+provides a copy of the environment
 variables in an immutable map. If you need a mutable map (modifiable copy) you
 can use `Map.from(Platform.environment)`.
 
-{% prettify dart %}
-Map environmentVars = Platform.environment;
+<?code-excerpt "misc/lib/tutorial/cmdline.dart (env)"?>
+```dart
+final envVarMap = Platform.environment;
 
-print('PWD = ${environmentVars["PWD"]}');
-print('LOGNAME = ${environmentVars["LOGNAME"]}');
-print('PATH = ${environmentVars["PATH"]}');
-{% endprettify %}
+print('PWD = ${envVarMap["PWD"]}');
+print('LOGNAME = ${envVarMap["LOGNAME"]}');
+print('PATH = ${envVarMap["PATH"]}');
+```
 
 `Platform` provides other useful properties that give
 information about the machine, OS, and currently
-running program.
-For example:
+running program. For example:
 
-* `Platform.isMacOS()`
-* `Platform.numberOfProcessors`
-* `Platform.script.path`
+- [Platform.isMacOS()]({{PlatformAPI}}/isMacOS.html){:target="_blank"}
+- [Platform.numberOfProcessors]({{PlatformAPI}}/numberOfProcessors.html){:target="_blank"}
+- [Platform.script]({{PlatformAPI}}/script.html){:target="_blank"}
```

Comment:
Notice that this last bullet now refers to `Platform.script` rather than `Platform.script.path`. In this way, we can link directly to the property's API page. Also, `script` has multiple properties of interest, though admittedly, in this case `path` is the most interesting. Let me know if you'd like this to refer to `Platform.script.path` as it did before.